.changeset/olive-elephants-protect.md

```diff
@@ -0,0 +1,17 @@
+---
+"@comet/admin": major
+---
+
+Remove `DialogContent` from `EditDialog`
+
+Add `DialogContent` as child of `EditDialog`.
```

Comment:
Here we should provide more context why the component was removed and when to add it again.